CHANGELOG.md

```diff
@@ -864,6 +864,7 @@
 * [CHANGE] Ruler: promote tenant federation from experimental to stable. #8400
 * [CHANGE] Ruler: promote `-ruler.recording-rules-evaluation-enabled` and `-ruler.alerting-rules-evaluation-enabled` from experimental to stable. #8400
 * [CHANGE] General: promote `-tenant-federation.max-tenants` from experimental to stable. #8400
+* [CHANGE] General: Unify "gogoproto/gogo.protochange" import in .proto files.  #11869
```

Comment:
Given this isn't a user-facing change, I don't think we need to list this in the changelog.